docs/cli-reference.md

```diff
@@ -61,6 +61,19 @@ cellery run wso2/cell2:1.0.1 -n mycell-2 --with-dependencies
 cellery run wso2/cell2:1.0.1 -n mycell-2 --with-dependencies --share-instances
 ```
 
+### Run with environment variables
+Pass environment variables to the Cell file's run method when starting the Cell. When starting a tree of dependencies, environment variables can be specified for each instance (not currently running) separately.
+Usage:
+```
+cellery run <ORGANIZATION_NAME>/<IMAGE_NAME>:<VERSION> -e <KEY>=<VALUE>
+cellery run <ORGANIZATION_NAME>/<IMAGE_NAME>:<VERSION> -l <ALIAS>:<INSTANCE> -e <INSTANCE>:<KEY>=<VALUE>
+```
+Example : 
+```
+cellery run wso2/cell1:1.0.1 -n mycell-1 -e 
```

Comment:
Is `-e` mandatory?

Reply:
No it's a mistake. Updated to `-e gcp.project=myproject` since this section is about Running with environment variables.